### PR TITLE
refac: Rename out_grid_area_code to neighbor_grid_area_code

### PR DESCRIPTION
**Pull request by johevemi**
# Description

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [ ] Tests are written and executed locally
- [x] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/databricks/calculation_engine/package/constants/energy_result_column_names.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 class EnergyResultColumnNames(ResultColumnNames):
     time_series_type = "time_series_type"
-    from_grid_area = "out_grid_area_code"
+    from_grid_area = "neighbor_grid_area_code"
```
Comment (JesperGomes) [inline source/databricks/calculation_engine/package/constants/energy_result_column_names.py:21]: rename variable name?

Comment (johevemi) [reply source/databricks/calculation_engine/package/constants/energy_result_column_names.py:21]: yes

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/SqlStatements/DeltaTableConstants/EnergyResultColumnNames.cs [hunk 2]
```diff
@@ -31,7 +31,7 @@ public class EnergyResultColumnNames
     public const string QuantityQualities = "quantity_qualities";
     public const string AggregationLevel = "aggregation_level";
     public const string TimeSeriesType = "time_series_type";
-    public const string FromGridArea = "out_grid_area_code";
+    public const string FromGridArea = "neighbor_grid_area_code";
```
Comment (JesperGomes) [inline source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/SqlStatements/DeltaTableConstants/EnergyResultColumnNames.cs:34]: variable name?

Comment (JesperGomes) [reply source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/SqlStatements/DeltaTableConstants/EnergyResultColumnNames.cs:34]: separate PR maybe?

File: source/databricks/calculation_engine/package/constants/energy_result_column_names.py (hunk 1 above)
Comment (JesperGomes) [reply source/databricks/calculation_engine/package/constants/energy_result_column_names.py]: or seperate PR?

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/SqlStatements/DeltaTableConstants/EnergyResultColumnNames.cs (hunk 2 above)
Comment (johevemi) [reply source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/SqlStatements/DeltaTableConstants/EnergyResultColumnNames.cs]: I renamed it now

File: source/databricks/calculation_engine/package/constants/energy_result_column_names.py (hunk 1 above)
Comment (johevemi) [reply source/databricks/calculation_engine/package/constants/energy_result_column_names.py]: I renamed it now

**Review by JesperGomes — APPROVED** [review]

**Pull request merged**